### DISCUSSION TITLE
Meter modbus ID, does it matter?

### DISCUSSION THREAD

**Discussion by cralwalker** [WillCodeForCats/solaredge-modbus-multi#566] [General]
I have found that my import/export meter have device ID 2. Does this have any impact in any way?

Meter 1, device ID 2   201/65536 import/export
Meter 2, device ID 1   201/65536 disabled
Meter 3, device ID 1   201/65536 disabled

**Comment by WillCodeForCats** [comment]
SolarEdge ships their meters pre-set to ID 2. It only matters if you have multiple meters on the meter bus.
